### DISCUSSION TITLE
Reprojecting

### DISCUSSION THREAD

**Discussion by marstonsward** [pangeo-data/xESMF#318] [Q&A] [answered]
Can xESMF regrid data onto another grid with another projection map. For example: latlon square grid onto a LAEA grid?

**Comment by aulemahal** [comment, marked as answer]
The answer is yes, but a nuance might help : xESMF (well ESMF really) does _regridding_, not _reprojecting_. As long as you have the latitude and longitude coordinates for your input and output grids, xESMF can do it. But, it does not provide tools to compute those coordinates. 

I guess you could generate the LAEA grid you want, then use something like `cartopy` (or directly `pyproj`) to get the lat/lon (`PlateCarree`) version of the coordinates and pass that to xESMF.
